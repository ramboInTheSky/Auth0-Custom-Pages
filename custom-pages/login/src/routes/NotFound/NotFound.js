import React from 'react'
import { browserHistory } from 'react-router'
import './NotFound.scss'

const goBack = (e) => {
  e.preventDefault()
  return browserHistory.goBack()
}

export const NotFound = () => (
  <div className="notFound">
    <h4>Something must have gone slightly wrong!</h4>
    <p><a href="#" onClick={goBack}>&larr; Back</a></p>
  </div>
)

export default NotFound
