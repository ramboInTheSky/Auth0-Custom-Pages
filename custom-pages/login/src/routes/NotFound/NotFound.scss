.notFound {
  padding: 20px;
  min-width: var(--col-width-mobile);
}
