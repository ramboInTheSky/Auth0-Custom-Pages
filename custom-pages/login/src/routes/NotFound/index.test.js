import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'
import NotFound from './'

Enzyme.configure({ adapter: new Adapter() })

describe('NotFound components', () => {
  it('should render <NotFound /> correctly', () => {
    expect(shallow(<NotFound.component />)).toMatchSnapshot()
  })
})
