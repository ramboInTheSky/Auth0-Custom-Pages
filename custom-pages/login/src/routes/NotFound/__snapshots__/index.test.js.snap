// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound components should render <NotFound /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4>
          Something must have gone slightly wrong!
        </h4>,
        <p>
          <a
            href="#"
            onClick={[Function]}
          >
            ← Back
          </a>
        </p>,
      ],
      "className": "notFound",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Something must have gone slightly wrong!",
        },
        "ref": null,
        "rendered": "Something must have gone slightly wrong!",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="#"
            onClick={[Function]}
          >
            ← Back
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "← Back",
            "href": "#",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "← Back",
          "type": "a",
        },
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4>
            Something must have gone slightly wrong!
          </h4>,
          <p>
            <a
              href="#"
              onClick={[Function]}
            >
              ← Back
            </a>
          </p>,
        ],
        "className": "notFound",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Something must have gone slightly wrong!",
          },
          "ref": null,
          "rendered": "Something must have gone slightly wrong!",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="#"
              onClick={[Function]}
            >
              ← Back
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "← Back",
              "href": "#",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "← Back",
            "type": "a",
          },
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFourteenAdapter {
      "options": Object {
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
          "getChildContext": Object {
            "calledByRenderer": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
